cmd/bosun/sched/template.go

```diff
@@ -295,26 +298,30 @@ func (c *Context) graph(v interface{}, filter bool) (interface{}, error) {
 			Filename:    name,
 			ContentType: "image/png",
 		})
-		return template.HTML(fmt.Sprintf(`<img alt="%s" src="cid:%s" />`,
+		return template.HTML(fmt.Sprintf(`<img alt="%s" src="cid:%s" />%s`,
 			template.HTMLEscapeString(fmt.Sprint(v)),
 			name,
+			footerHTML,
 		)), nil
 	}
-	if err := c.schedule.ExprSVG(nil, &buf, width, height, res, title, time.Now().UTC()); err != nil {
+	if err := c.schedule.ExprSVG(nil, &buf, width, height, unit, res); err != nil {
 		return nil, err
 	}
+	buf.WriteString(footerHTML)
 	return template.HTML(buf.String()), nil
 }
 
 // Graph returns an SVG for the given result (or expression, for which it gets the result)
 // with same tags as the context's tags.
-func (c *Context) Graph(v interface{}) (interface{}, error) {
-	return c.graph(v, true)
+func (c *Context) Graph(v interface{}, unit ...string) (interface{}, error) {
```

Comment:
When would I pass multiple unit args? Is this just vararg to make it a non-breaking change? I would rather break people than use var args as optional parameters.